<?xml version="1.0"?>
<launch>

   <rosparam command="load" file="$(find manipulator_grasping)/config/vs_grasping_config.yaml" />
   
   <node pkg="manipulator_grasping" name="vs_grasping_node" type="vs_grasping_node" output="screen" />

</launch>